docs/articles/utilities/logging.md

```diff
@@ -1,11 +1,12 @@
 ---
 uid: logging
-title: Logging
+title: Logging in Akka.NET
 ---
 
 # Logging
 
-For more info see real Akka's documentation: <http://doc.akka.io/docs/akka/2.0/scala/logging.html>
+> ![NOTE]
+> For info
```

Comment:
Empty documentation?

Reply:
Looks like I got interrupted mid-sentence. I'll go finish that.